templates/default/custom.properties.erb

```diff
@@ -221,34 +221,36 @@ org.opencastproject.solr.dir=${org.opencastproject.storage.dir}/solr-indexes
-# This setting is configured here and not in the OAI-PMH server's config since it is shared amongst several
-# OAI-PMH related components.
-#org.opencastproject.oaipmh.mountpoint=/oaipmh
-
 # Whether to accept a job whose load exceeds the host's max load
 # Default: true
 #org.opencastproject.job.load.acceptexceeding=true
 
-# The number of times to retry a request if the nonce expires.
-#org.opencastproject.security.digest.nonce.retries=12
+######### Capture and Ingest #########
 
-# The configuration property specifying the minimum amount of time in seconds wait before retrying a request after
-# a nonce timeout. Default is 300 seconds (5 minutes).
-#org.opencastproject.security.digest.nonce.base.time=300
+# Timeout for capture agent status, in minutes.
+# Capture agents which have not sent status updates for this period will be marked as offline.
+# Default: 120 minutes (2 hours)
+#org.opencastproject.capture.admin.timeout=120
 
-# The maximum amount of time to wait in addition to the base time for a random generator to add after a nonce timeout
-# so that requests that timeout won't all try again at exactly the same time. Default is 300 seconds (5 minutes).
-#org.opencastproject.security.digest.nonce.variable.time=300
+# The ID of the default workflow definition to run when media are ingested
+#org.opencastproject.workflow.default.definition=schedule-and-upload
 
-# This changes the number of seconds from when an internal request is made until a signed URL will expire. More
-# specifically, the HTTP client needs access to internal storage areas such as the working file repository as well as to
-# distributed artifacts on the downloads and streaming servers, all of which are protected by verification components.
-# Default is 60 seconds as it shouldn't take longer than that to make a request to a server. This will have no impact on
-# a system where url signing is not configured. For more information please see:
-# http://docs.opencast.org/latest/admin/configuration/stream-security/#configuration-of-url-signing-timeout-values
-#org.opencastproject.security.internal.url.signing.duration=60
+# #DCE only config: org.opencastproject.file.repo.url? It's not in 5x dev Mar 2018
+# The base URL of the file server.  When using a shared filesystem between servers, set all servers to use the same URL.
+# Only then will hard linking between the working file repository and the workspace be enabled to prevent downloads.
+#DCE Rute: with 5.x this is gotten from the mh_default_org configuration instead? To do: confim.
+org.opencastproject.file.repo.url=<%= @admin_url %>
+
+# The max number of ingests to allow at the same time. If more ingests try than the max they will receive service
+# unavailable. A value of 0 means that the server will accept all ingests.
+# Default: 0
+#org.opencastproject.ingest.max.concurrent=0
+
+# The maximum number of concurrent files to ingest from the inbox directory
+# Default: 1
+#org.opencastproject.inbox.threads=1
```

Comment:
Do we use the inbox functionality enough that allowing multiple concurrent ingests helps anything?

Reply:
We are not currently using it (it was used a lot for repubs in the past).